mestopy/mestopy.py

```diff
@@ -1 +1,284 @@
-"""Main module."""
+from scipy.signal import oaconvolve
+from pyfar import Signal
+
+
+# Class to generate ref-Objects, that can bei part of the MeasurementChain
+class Device(object):
+    """Class for device in MeasurementChain.
+
+    This class holds methods and properties of a device in the
+    'MeasurementChain' class. A device can be e.g., a sound card or a
+    pre-amplifier, described by a frequency response and/or sensitivity.
+    """
+
+    def __init__(self, name, data=None, sens=1, unit=None):
+        """Init Device with data.
+
+        Attributes
+        ----------
+        name : str
+            Name of the device.
+        data : Signal
+            Signal data that reprensets the inversed frequency response of
+            the device. Data can be in domain 'freq' or 'time' and will be
+            transformed if necessary. The default is None, in this case a
+            perfect flat frequency response is assumed and only sensitivity as
+            a factor is applied.
+            Caution: Make sure to use frequency responses without artefacts,
+            as they will influence the quality of compensations calculated with
+            the measurement chain. Most ideal this should be regularized
+            inversed frequency responses.
```

Comment:
Maybe more explicit:
```suggestion
            Caution: Avoid large gains in the frequency responses because they will boost measurement noise and might cause numerical instabilities. One possibility to avoid this is to use regularized inversion.
```